content/quickstart/gcp/configure.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Configure Google Cloud
+weight: 4
+menu:
+  quickstart:
+    parent: gcp
+    identifier: gcp-configure
+---
+
+<!-- TODO inline a streamlined version of configuring the cloud here. -->
+
+<a href="{{< relref "/reference/clouds/gcp/setup.md" >}}" target="_blank">Configure Google Cloud</a> so the Pulumi CLI can connect to Google Cloud.
```

Comment:
Similar to earlier comments on AWS, Azure, say that if `gcloud` is configured, this will just work.

Reply:
Fixed